plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/configs/BuildLimitsConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.programmerdan.minecraft.simpleadminhacks.configs;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.SimpleHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.utilities.BuildLimit;
+import org.bukkit.configuration.ConfigurationSection;
+
+import java.util.List;
+
+public final class BuildLimitsConfig extends SimpleHackConfig {
+	private boolean enabled;
+	private BuildLimit[] buildLimits;
+
+	public BuildLimitsConfig(SimpleAdminHacks plugin, ConfigurationSection base) {
+		super(plugin, base);
+	}
+	public BuildLimitsConfig(ConfigurationSection base) {
+		super(SimpleAdminHacks.instance(), base);
+	}
+
+	@Override
+	protected void wireup(ConfigurationSection config) {
+		this.enabled = config.getBoolean("enabled");
+
+		if (this.enabled) {
+			List<?> rawList = config.getList("limits");
+			if (rawList != null && rawList.size() > 0) {
+				try {
+					this.buildLimits = rawList.toArray(new BuildLimit[rawList.size()]);
```

Comment:
Instead of try-catching an assignment, you could use an iterator to check that each element is an instance of `BuildLimit`, which then lets you print nice warnings for each offending entry.